src/scripts/towns/TownList.ts

```diff
@@ -5870,6 +5870,14 @@ const Lysandre3 = new NPC('Lysandre', [
     requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 12), new QuestLineStepCompletedRequirement('A Beautiful World', 14, GameConstants.AchievementOption.less)]),
 });
 
+const SantaluneSageChen = new NPC('Sage Chen', [
+    'Ever since growing old I\'m no longer the Pokémon Trainer that I used to be. Now I always like to keep my garden full of Kasib Berries.',
+    'One day, I forgot to harvest them, and within its withered husk, I found a small stone that enabled my Banette to perform a Mega Evolution."',
+], {
+    image: 'assets/images/npcs/Sage.png',
+    requirement: new ObtainedPokemonRequirement('Banette', true),
```

Comment:
```suggestion
    requirement: new ObtainedPokemonRequirement('Banette'),
```
The `true` makes it so that you have to _not_ have a Banette for the npc to show up.